### PR TITLE
Prepare release 2.4.2

### PR DESCRIPTION
**Pull request by abikouo**
Prepare Release 2.4.2

### CODE REVIEW

**Review by jillr — APPROVED** [review]

File: tests/integration/targets/helm_kubeconfig/tasks/main.yml [hunk 1]
```diff
@@ -1,19 +1,21 @@
 ---
 - name: Test helm with in-memory kubeconfig
   include_tasks: "from_in_memory_kubeconfig.yml"
+  loop_control:
+    loop_var: test_helm_version
+  with_items:
+    - "v3.10.3"
```
Comment (alinabuzachis) [inline tests/integration/targets/helm_kubeconfig/tasks/main.yml:7]: Is this change intended in this PR?

Comment (abikouo) [reply tests/integration/targets/helm_kubeconfig/tasks/main.yml:7]: Yes it is. The reference is here https://github.com/ansible-collections/kubernetes.core/pull/670#issuecomment-1947945547
